src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -2919,6 +2919,83 @@ private static OtherClass InitializeProperty()
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [WorkItem(12708, "https://github.com/dotnet/roslyn/issues/12708")]
+        public async Task TestNoOverloadOnEventAssignment()
```

Comment:
I would call this something like `TestWithIncompatibleOverloadOnEventAssignment`.  The `No` part makes me think that you are testing that it *doesn't* show up.